conf/maven/arlas-client-deploy-setting.xml

```diff
@@ -0,0 +1,10 @@
+<distributionManagement>
+    <snapshotRepository>
```

Comment:
Should we publish snapshot to the same repository as release? Do we want to make snapshot public?

Reply:
The release process does not publish the snapshot version
The url of snapshot is the public package too and it's specified, just in case